src/main/java/org/stocker/PingVerticle.java

```diff
@@ -99,15 +100,23 @@ public void start() {
      */
     private void populateDB(UpdateRecordsRoute updateRecordsRoute) {
         LocalDate localDate = LocalDate.now();
-        for(int i=0 ; i < 10 ; i++){
+        for(int i=0 ; i < 10 ; i++) {
             Date date = localDate.minusDays(i).toDate();
-            try {
-                updateRecordsRoute.updateStocksForDate(date);
-            } catch (StockDataNotFoundForGivenDateException e) {
-                logger.warning(" failed for date - " + date.toString());
-            } catch (NseDataObjParseException e) {
-                logger.warning(" failed with parsing error for date - " + date.toString());
-            }
+            updateStocksForDate(updateRecordsRoute, "IN", date);
```

Comment:
just do one or other . In case of same symbols between these 2 markets , this just produces uncertainty today .  Lets stick to one market .  Hardcoded . Make sure our stuff works . Moving to multiple markets will be a natural extension ( since today most business logic doesn't depend on it , just the part of getting stock data does )